tests/test_gen_cuda_deps.py

```
@@ -0,0 +1,339 @@
+        CUDA_DEPS.replace('{ name = "accelerate" }', '{ name = "accelerate", specifier = "=>1" }'), encoding="utf-8"
+    )
+
+    with pytest.raises(Exception, match="Invalid specifier"):
+        GENERATOR.load_cuda_deps_config(config_path)
+
+
+def test_build_cuda_pyproject_fragment_rejects_invalid_raw_requirement(tmp_path: Path) -> None:
+    config_path = tmp_path / "cuda_deps.toml"
+    config_path.write_text(CUDA_DEPS.replace('"faker"', '"not @@@ invalid"'), encoding="utf-8")
+
+    with pytest.raises(Exception):
```

Ruff (0.15.13)

[warning] Do not assert blind exception: `Exception`

(B017)